### DISCUSSION TITLE
How do I modify the pagination for a specific type of content?

### DISCUSSION THREAD

**Discussion by matthieu2607** [Cecilapp/Cecil#1473] [Q&A] [answered]
Hi @ArnaudLigny,

My question is in the title :)

I don't know if it's possible?

Best regards,

Matthieu

**Comment by ArnaudLigny** [comment, marked as answer]
Hello @matthieu2607 , what do you mean by "modify"?

You can have a different number of pages and path by section, by creating an `index.md` file at root of the section, for example:

`/my-section/idnex.md`:

```yaml
---
pagination:
  max: 5
  path: "page"
---
```

See https://cecil.app/documentation/content/#pagination.

**Comment by matthieu2607** [comment]
Big thanks @ArnaudLigny
